mimic/model/maas_objects.py

```diff
@@ -600,6 +603,55 @@ def get_host_info(self, available_types, requested_types, entity_id, clock):
                            current_timestamp)))
                      for rtype in requested_types])
 
+    def list_connections(self):
+        """
+        Lists connections that this agent has.
+        """
+        return [AgentConnection(id=self.id + '_' + dc,
+                                guid=self.connection_guid,
+                                agent_id=self.id,
+                                endpoint=dc,
+                                datacenter=dc)
+                for dc in ('dfw1', 'ord1', 'lon3')]
+
+
+@attributes([Attribute('id', instance_of=text_type),
```

Comment:
I'm actually working on a PR upstream to redo this module in terms of `attrs`, but wanted to get the feature out first.